metric.go

```diff
@@ -129,40 +118,27 @@ func makeBuckets(m *dto.Metric) map[string]string {
 	return result
 }
 
-func fetchMetricFamilies(url string, ch chan<- *dto.MetricFamily, certificate string, key string) {
+// FetchMetricFamilies does stuff
```

Comment:
`// FetchMetricFamilies retrieves metrics from the provided URL, decodes them into MetricFamily proto messages, and sends them to the provided channel. It returns after all MetricFamilies have been sent.`